src/main.cpp

```diff
@@ -202,8 +202,20 @@ main(int argc, char *argv[])
     QCommandLineParser parser;
     parser.addHelpOption();
     parser.addVersionOption();
-    QCommandLineOption debugOption(QStringLiteral("debug"), QStringLiteral("Enable debug output"));
+    QCommandLineOption logLevel(
+      QStringLiteral("loglevel"),
+      QStringLiteral("Set the log level, or a <component>=<level> list like \"warn,ui=off\". "
+                     "levels:{trace,debug,info,warning,error,critical,off} "
+                     "components:{crypto,db,net,qml,ui}"),
```

Comment:
There is actually one more, "mtx" or so from the mtxclient lib

Reply:
Thanks. I've added it in my local copy & will include it with my next update.

Reply:
Actually, from a quick look at the mtxclient code, I don't think it adds its logger to [spdlog's registry](https://github.com/gabime/spdlog/wiki/5.-Logger-registry#registering-new-loggers). Did I miss something there? If not, there's no point in mentioning "mtx" in our help text, since spdlog will ignore it.

Reply:
I'll try registering mtxclient's logger from within Nheko. I was concerned at first about what would happen if mtxclient starts registering its own logger in some future version, but it turns out there's a [documented exception](https://github.com/gabime/spdlog/wiki/5.-Logger-registry#registry-conflicts) for that, so I think I can handle it safely.

Reply:
I didn't know about registering loggers. I think we should just not register the mtxclient logger in Nheko and just change that in mtxclient to always register it. But it makes sense to already document it here, imo.

Reply:
Whose project is mtxclient? Is this change something should be done upstream?

Reply:
https://github.com/Nheko-Reborn/mtxclient is the upstream

Reply:
Oh! Interesting. I thought most Matrix clients used a library from matrix.org, or else a stand-alone project. I didn't expect Nheko to have a custom one.